### PR TITLE
Set the "Accountable Person" dropdown value when editing an item

### PR DESCRIPTION
**Pull request by forevermatt**
### Fixed
- Set the "Accountable Person" dropdown value when editing an item

### CODE REVIEW

**Review by hobbitronics — APPROVED** [review]
I am approving this, but one of us will have to sort out some merge conflicts. How do you want to proceed?

File: src/components/forms/ItemForm.svelte [hunk 1]
```diff
@@ -93,6 +103,7 @@ const saveForLater = (event: Event) => {
 }
 
 const setInitialValues = (item: PolicyItem) => {
+  accountablePersonId = item.accountable_user_id || item.accountable_dependent_id || accountablePersonId
```
Comment (hobbitronics) [inline src/components/forms/ItemForm.svelte:106]: I don't think this is doing anything. I think the magic is happening in onAccountablePersonSelectPopulated

**Comment by forevermatt** [comment]
Waiting for #165 to be completed before moving this forward

**Comment by forevermatt** [comment]
All improvements superseded by #165

**Pull request closed without merging**